spec/support/jasmine-browser.json

```diff
@@ -0,0 +1,19 @@
+{
+  "srcDir": "public/assets",
+  "srcFiles": [
+    "application-*.js"
+  ],
+  "cssFiles": [
+    "application-*.css"
+  ],
+  "specDir": "spec/javascripts",
+  "specFiles": [
+    "**/*[sS]pec.js"
+  ],
+  "helpers": [
+    "https://unpkg.com/jasmine-jquery@2.1.1/lib/jasmine-jquery.js",
+    "https://unpkg.com/fetch-mock@9.11.0/es5/client-bundle.js",
+    "helpers/*.js"
+  ],
+  "browser": "headlessChrome"
+}
```

Comment:
Nit: missing new line

Reply:
Visual Studio Code insists on removing the final newline from JSON files – [apparently deliberately](https://github.com/microsoft/vscode/issues/96112). But I'll use a different editor to add that newline!